src/timely-util/src/containers/stack.rs

```diff
@@ -25,7 +26,7 @@ use crate::containers::array::Array;
 static ENABLE_CHUNKED_STACK: AtomicBool = AtomicBool::new(false);
 
 /// A runtime-configurable wrapper around timely stacks and chunked stacks.
-#[derive(Clone)]
+#[derive(Clone, Serialize, Deserialize)]
```

Comment:
The serde implementation for `TimelyStack` is entirely untested. Could you check it to increase our confidence it does the right thing?

Reply:
I did! In fact, it is identical to the one I implemented here (because I copied it)